Source/ModulePhysics.cpp

```diff
@@ -0,0 +1,43 @@
+#include "ModulePhysics.h"
+#include "Bullet/btBulletDynamicsCommon.h"
+
+ModulePhysics::ModulePhysics()
+{
+}
+
+ModulePhysics::~ModulePhysics()
+{
+}
+
+bool ModulePhysics::Init()
+{
+	//collision configuration contains default setup for memory, collision setup. Advanced users can create their own configuration.
+	btDefaultCollisionConfiguration* collisionConfiguration = new btDefaultCollisionConfiguration();
```

Comment:
All the uses of `new` inside this method don't have the accompaning `delete`s, which means the allocated memory is never freed. If we don't need to save this objects as members, can't we just use references?
```suggestion
	btDefaultCollisionConfiguration collisionConfiguration;
```
And if we need to save them, consider using unique pointers.

Reply:
Yeah I agree, but this code is meerly to check the integration of the library it is not intented to stay as is.  It is the HelloWorld provided by the library and the only reason it is in this PR is to check that it compiles in more than one machine (mine).

Reply:
Ah whoops, didn't realize this wasn't pointed to `develop`. Carry on then! :P